proposals/0186-transaction-data-size-specification.md

```diff
@@ -0,0 +1,152 @@
+---
+simd: '0186'
+title: Transaction Data Size Specification
```

Comment:
This title reads as if the proposal specifies the size of a serialized transaction rather than size of the account data loaded for processing a transaction. How about `Transaction Account Data Size Specification`?

Reply:
went with `Loaded Transaction Data Size Specification`

Reply:
went with `Loaded Transaction Data Size Specification`